exercises/affine-cipher/.meta/.version

```diff
@@ -0,0 +1 @@
+2
```

Comment:
The first published version should always be 1.

Reply:
Okay, so do I just change that by hand then, since the --update flag on `bin/generate` wants to go to 2 now?

Or is there some way to generate it to one in this case?